broker/amqp.go

```diff
@@ -157,6 +187,26 @@ func (a *AMQP) Subscribe(event string) (err error) {
 	return
 }
 
+func (a *AMQP) Call(event string, opts amqp.Publishing) ([]byte, error) {
+	correlation := uuid.New().String()
```

Comment:
It may be more performant to keep the byte array representation for comparison purposes later.